pkg/controller/namespace/namespace_controller.go

```diff
@@ -355,13 +355,24 @@ func (r *ReconcileNamespace) checkAndCreateRoleBindings(namespace *corev1.Namesp
 // Create openpitrix runtime
 func (r *ReconcileNamespace) checkAndCreateRuntime(namespace *corev1.Namespace) error {
 
+	op, err := cs.ClientSets().OpenPitrix()
```

Comment:
we shouldn't use clienset any more, it's anti pattern, bad for unit testing. Instead, we should pass op dependency to controller.

Reply:
I will create a new PR to refactor namespace controller.